src/features/comment/update-comment/ui/UpdateCommentForm.tsx

```diff
@@ -0,0 +1,42 @@
+import {
+  Button,
+  DialogContent,
+  DialogHeader,
+  DialogTitle,
+  Textarea,
+} from '../../../../shared/ui/components';
+import { useDialogStore } from '../../../../shared/hook/useDialogStore';
+import { IComment } from '../../../../entities/comment/model/type';
+import { useUpdateComment } from '../model/useUpdateComment';
+
+interface UpdateCommentForm {
+  comment: IComment;
+}
+
+const UpdateCommentForm = ({ comment }: UpdateCommentForm) => {
+  const { setCommentModal } = useDialogStore();
+  const { editComment, setBody, updateComment } = useUpdateComment(
+    comment,
+    () => {
+      setCommentModal({ show: false, content: null });
+    }
+  );
```

Comment:
좋은 것 같아요, 근데 쓰는 쪽에서 넘기는 함수가 onSuccess 인지 알면 더 좋을 것 같아서 이건 어떠신가요

```suggestion
  const { editComment, setBody, updateComment } = useUpdateComment(
    comment,
    {
      onSuccess: () => setCommentModal({ show: false, content: null }),
    }
  );
```